dotcom-rendering/src/web/components/Callout/CheckboxSelect.tsx

```diff
@@ -39,6 +45,7 @@ export const CheckboxSelect = ({ formField, formData, setFormData }: Props) => (
 						label={option.label}
 						value={option.value}
 						checked={isCheckboxChecked}
+						error={validationErrors?.[formField.id] ? true : false}
```

Comment:
The `CheckboxSelect` only takes a boolean as an error, so if an error is present we say true, if not false.

Reply:
The checkbox group takes a string error if you wanted to display the validation error as well as get the checkbox styling

Reply:
Thanks - have now pushed a change that passes the error there too.